src/NiivueMutator.ts

```diff
@@ -19,6 +21,8 @@ class NiivueMutator {
 
   constructor(nv: Niivue) {
     this.nv = nv;
+    // @ts-ignore
+    window.nv = nv;
```

Comment:
Seems like this was for debugging and accidentally left in. Could you please remove it?

Reply:
Done.